src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalAmount -= 20;
+  } else if (days >= 7) {
+    totalAmount -= 50;
+  }
+
+  return totalAmount;
```

Comment:
use early return pattern
```suggestion
  if (days >= 3 && days < 7) {
   return totalAmount - 20;
  } 
  
   if (days >= 7) {
    return totalAmount - 50;
  }

```